core/utils.js

```diff
@@ -19,13 +19,13 @@
 
   }
 
-  function decodeBase64(d) {
-    return Buffer.from(d,'base64').toString('binary');
-  }
+  // function decodeBase64(d) {
+  //   return Buffer.from(d,'base64').toString('binary');
+  // }
 
-  function encodeBase64(d) {
-    return Buffer.from(d,'binary').toString('base64');
-  }
+  // function encodeBase64(d) {
+  //   return Buffer.from(d,'binary').toString('base64');
+  // }
```

Comment:
These were not called from anywhere, wasn't sure if they needed to be exported for use in case, or just removed?